core/components/atoms/popperWrapper/PopperWrapper.tsx

```diff
@@ -377,15 +366,15 @@ export class PopperWrapper extends React.Component<PopperWrapperProps, PopperWra
       }
 
       const popperAnimationStyles = {
-        transformOrigin: this.positionOffset[this.props.placement],
         animation: open
-          ? `popper-open-${uniqueKey} 120ms cubic-bezier(0, 0, 0.38, 0.9)`
-          : `popper-close-${uniqueKey} 120ms cubic-bezier(0.2, 0, 1, 0.9)`,
+          ? `popper-open-${uniqueKey} 120ms cubic-bezier(0, 0, 0.38, 0.9), popper-fade-in 120ms`
+          : `popper-close-${uniqueKey} 120ms cubic-bezier(0.2, 0, 1, 0.9) 120ms, popper-fade-out 120ms 120ms`,
       };
 
       childrenStyles = {
         ...childrenStyles,
         ...popperAnimationStyles,
+        overflow: 'hidden',
```

Comment:
What is the need of overflow:'hidden here

Reply:
because the children of the popover are on top of its parent when parent transition happens children need to have overflow hidden.